Client/Frontend/Browser/Tabs/TabTableViewCell.swift

```diff
@@ -38,22 +50,32 @@ class TabTableViewCell: UITableViewCell, Themeable {
         }
         
         websiteTitle.numberOfLines = 2
```

Comment:
<img width="402" alt="image" src="https://user-images.githubusercontent.com/8919439/101539175-16ff1880-396c-11eb-982a-a40430e33a83.png">

This will fix constraint issue for 2 lines